verify_project.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -e
 
-py.test --cov=demo_python_at
+py.test --cov=demo_python_at demo_python_at
```

Comment:
@extsoft Changes applied:
- Updated the commit message to describe the problem and it's solving correctly. 
- Modified `verify_projects.sh` to runs unit tests only.